src/scalmat.jl

```diff
@@ -92,10 +89,10 @@ end
 
 function X_invA_Xt(a::ScalMat, x::StridedMatrix)
     @check_argdims dim(a) == size(x, 2)
-    lmul!(a.inv_value, x * transpose(x))
+    ldiv!(a.value, x * transpose(x))
```

Comment:
It seems `ldiv!(::Number, ::AbstractArray)` is not available on Julia 1.0. I did not realize this since there is no note in the docstring: https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/#LinearAlgebra.ldiv!